homeassistant/bootstrap.py

```diff
@@ -335,10 +335,13 @@ def _get_domains(hass: core.HomeAssistant, config: Dict[str, Any]) -> Set[str]:
         await asyncio.wait(futures.values())
         errors = [domain for domain in domains if futures[domain].exception()]
         for domain in errors:
+            exception = futures[domain].exception()
+            kwargs = {}
+            kwargs["exc_info"] = (type(exception), exception, exception.__traceback__)
             _LOGGER.error(
-                "Error setting up integration %s - received exception %s",
+                "Error setting up integration %s - received exception",
                 domain,
-                futures[domain].exception(),
+                **kwargs,  # type: ignore
```

Comment:
Why do it like this and not do `exc_info=(type(exception), exception, exception.__traceback__)` ?

Reply:
i just copied it from the default exception handler in core.py, but you are correct. i made the change. do you want me to change the one in core.py as well?

Reply:
BTW, if i understand the python behavior correctly, the one in core.py can also be replaced by exc_info=True, as it happens during the context of the exception, but not sure as i am new to python

Reply:
Yes, if we're within a `catch` block, we should let the logger get it themselves.